### ISSUE TITLE
[Bug] NameError on startup (wxMac) [sf#225]

### ISSUE THREAD

**Issue by DietmarSchwertberger** [DietmarSchwertberger/wxGlade#296]
**Reported by hraban on 2017-12-18 08:09 UTC**
Looks like you didn’t test on wxMac. It works if I use the wx.IDs from the code block above.
~~~
ERROR   : None
An unexpected error occurred!

Exception type:      <type 'exceptions.NameError'>
Exception details:   global name 'ABOUT_ID' is not defined

Application stack traceback:
  File "wxglade.py", line 249, in <module>
    run_main()
  File "wxglade.py", line 244, in run_main
    main.main(options.filename)
  File ".../workspace/wxglade/main.py", line 1083, in main
    app = wxGlade()
  File "/usr/local/lib/wxPython-3.0.0.0/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 8631, in __init__
    self._BootstrapApp()
  File "/usr/local/lib/wxPython-3.0.0.0/lib/python2.7/site-packages/wx-3.0-osx_cocoa/wx/_core.py", line 8196, in _BootstrapApp
    return _core_.PyApp__BootstrapApp(*args, **kwargs)
  File ".../workspace/wxglade/main.py", line 984, in OnInit
    frame = wxGladeFrame()
  File ".../workspace/wxglade/main.py", line 215, in __init__
    self.create_menu(parent)
  File ".../workspace/wxglade/main.py", line 403, in create_menu
    wx.App_SetMacAboutMenuItemId(ABOUT_ID)

Date and time:       2017-12-18T09:01:06.617156
Python version:      2.7.13
wxPython version:    3.0.0.0
wxWidgets platform:  __WXMAC__
wxGlade version:     0.8.0a9
~~~

**Comment by DietmarSchwertberger** [comment]
**Updated by dschwertberger on 2017-12-18 18:00 UTC**
- **status**: open --> accepted
- **assigned_to**: Dietmar Schwertberger

**Comment by DietmarSchwertberger** [comment]
**Commented by dschwertberger on 2017-12-18 18:00 UTC**
Thanks for reporting.
I have only been testing Python 3.6 and Phoenix. I probably need to install one 3.0 or 2.8 as well.

**Comment by DietmarSchwertberger** [comment]
**Updated by dschwertberger on 2017-12-18 19:08 UTC**
- **status**: accepted --> pending

**Comment by DietmarSchwertberger** [comment]
**Commented by dschwertberger on 2017-12-18 19:08 UTC**
I have just installed 3.0.2 on Python 2.7.
With your changes being in the repository now, the test cases are passing.
My recommendation is to use Phoenix, though.

**Comment by DietmarSchwertberger** [comment]
**Updated by dschwertberger on 2017-12-18 19:15 UTC**
- **status**: pending --> closed

**Comment by DietmarSchwertberger** [comment]
**Commented by dschwertberger on 2017-12-18 19:15 UTC**
Fixes are in 0.8.0a13, which is online now.
